jobs/jobs.go

```
@@ -105,7 +106,8 @@ func (mgr *JobManager) AddJob(j Job) {
 
 func (mgr *JobManager) recovery() {
 	if r := recover(); r != nil {
-		mgr.tryCacheError(fmt.Errorf("job panicked: %s", r))
+		err := fmt.Errorf("job panicked: %s", errors.Wrap(r, 0).ErrorStack())
```

similar to tm-events, I think you are suppressing the error message here